### PR TITLE
Add SYSTEM_PLATFORM_UWP infoLabel

### PR DESCRIPTION
**Pull request by enen92**
This pull request adds a `SYSTEM_PLATFORM_UWP` infolabel for UWP similar to what we have for other platforms.

## Description
`SYSTEM_PLATFORM_UWP` is mapped to `TARGET_WINDOWS_STORE` so that python addons can check if the addon is running on uwp/xbox via `xbmc.getCondVisibility("system.platform.uwp")`.

## Motivation and Context
Currently there is no builtin way for addons and skins to check if it is a uwp build. Both uwp (TARGET_WINDOWS_STORE) and generic windows build (TARGET_WINDOWS_DESKTOP) will return true to `xbmc.getCondVisibility("system.platform.windows")`. Addon devs are starting to implement custom checks to detect UWP (e.g. https://github.com/xbmc/repo-plugins/pull/1802/files#diff-2af6d5cdfb6cf7431d3675f9a4251e3fR61) as a way to prevent known errors/block sections while still allowing access to the addon and some of its features.

## How Has This Been Tested?
Made a simple addon to present a Dialog.ok() with the result of `xbmc.getCondVisibility("system.platform.uwp")`. Received false as expected on OSX.
I don't have any windows machine nor xbox so this needs to be tested for "true" in those systems. You can find the testing addon here: https://github.com/enen92/script.testuwp

## Screenshots (if appropriate):
On OSX:

Thanks to @afedchin for the help
@Lunatixz for reference

Requires addition to the wik if accepted: https://kodi.wiki/view/List_of_boolean_conditions

## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [X] My change requires a change to the documentation, either Doxygen or wiki
- [X] I have updated the documentation accordingly
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [X] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: xbmc/guilib/guiinfo/GUIInfoLabels.h [hunk 1]
```diff
@@ -418,7 +418,7 @@
 #define SYSTEM_PLATFORM_DARWIN      743
 #define SYSTEM_PLATFORM_DARWIN_OSX  744
 #define SYSTEM_PLATFORM_DARWIN_IOS  745
-//#define SYSTEM_PLATFORM_DARWIN_ATV2 746
```
Comment (Rechi) [inline xbmc/guilib/guiinfo/GUIInfoLabels.h]: remove or leave it untouched

**Comment by afedchin** [comment]
> `SYSTEM_PLATFORM_UWP` is mapped to `TARGET_WINDOWS_STORE` so that python addons can check if the addon is running on uwp/xbox via `xbmc.getCondVisibility("system.platform.uwp")`.

python add-ons can check the same in other way `sys.platform == "win10"`. `win10` means that a script is run on uwp platform PC, Xbox or Mobile.

**Comment by afedchin** [comment]
but I agree that `System.Platform.UWP` info is required, it can be used in skins.

**Review by afedchin — APPROVED** [review]

**Comment by enen92** [comment]
bump. Anything that still needs to be done here?

**Comment by Rechi** [comment]
@enen92 I assumed you have commit rights in this repo.

jenkins build and merge

**Comment by enen92** [comment]
@Rechi Thanks. Can you please add the wiki-needed label as well?

**Pull request merged**